lib/modules/platform/gitea/readme.md

```diff
@@ -38,7 +38,14 @@ If you use Gitea packages, add the `read:packages` scope.
 
 - none
 
-## Repo autodiscover sorting
+## Repo autodiscover
+
+Renovate can discover repositories on Gitea using the autodiscover feature.
+Repositories are ignored when one of the following conditions is met:
+
+- The repository is a `mirror`
+- We don't have push or pull permissions to that repository.
+- Pull requests are disabled for that repository.
```

Comment:
```suggestion
- The repository is a `mirror`
- We do not have push or pull permissions to that repository
- Pull requests are disabled for that repository
```

We don't use punctuation at the end of list items.
Also I think we want to expand contractions like `don't` to `do not`.